frontend/src/components/Root/Root.tsx

```diff
@@ -0,0 +1,211 @@
+          { title: 'Other', value: otherTotal, color: '#86E0FF' },
+        ];
+      }
+      setPieChartData(pieData);
+    };
+
+    const fetchSchedule = async () => {
+      let rows: DataDisplay[] = [];
+      let temp_id = 1;
+      const notifications = await getAllNotificationsByCurrentUser();
+      notifications.data.forEach((ele: any) => {
+        rows.push({
+          id: temp_id,
+          date: ele.date.split('T')[0],
+          type: ele.type,
+          description: ele.message,
+        });
+        temp_id++;
+      });
+      setScheduleRows(rows);
+    };
+
+    loadPieChartData();
+    fetchSchedule();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <div className={className.root}>
+      <Grid
+        container
+        spacing={0}
+        direction="column"
+        justifyContent="center"
+        alignItems="center"
+        style={{ minHeight: '100vh' }}
+      >
+        <Grid container item xs={12}>
+          <Grid item xs={1}></Grid>
+          <Grid item xs={3}>
+            <Card elevation={3} className={className.topRow}>
+              <CardHeader
+                avatar={<PieChartOutlined fontSize="large" color="primary" className={className.ToolBarIcon} />}
+                title={<Typography variant="h6">Expenses</Typography>}
+              />
+              <Divider />
+              <CardContent style={{ backgroundColor: '#fff', padding: 24 }}>
+                <PieChart
+                  data={pieChartData}
+                  radius={PieChart.defaultProps.radius - 7}
+                  label={({ dataEntry }) => dataEntry.title + ' (' + Math.round(dataEntry.percentage) + '%)'}
+                  lineWidth={30}
+                  labelPosition={50}
+                  labelStyle={(index) => ({
+                    fill: pieChartData[index].color,
+                    fontSize: '5px',
+                  })}
+                />
+              </CardContent>
+            </Card>
+          </Grid>
+          <Grid item xs={7}>
+            <Paper elevation={3} className={className.topRow}>
+              <Grid container item direction="column" xs={12} spacing={0} style={{ height: 560, width: '100%' }}>
+                <Grid item>
+                  <AppBar position="static" color="transparent" elevation={0}>
+                    <Toolbar className={className.ToolBar}>
+                      <ScheduleIcon fontSize="large" color="primary" className={className.ToolBarIcon} />
+                      <Typography className={className.ToolBarTitle} variant="h6">
+                        Notifications
+                      </Typography>
+                    </Toolbar>
+                  </AppBar>
+                </Grid>
+                <Grid item xs>
+                  <DataGrid
+                    rows={scheduleRows}
+                    columns={columns}
+                    pageSize={20}
+                    // components={{
```

Comment:
Remove commented line

Reply:
Also the content doesn't seem to scale well on larger displays:
![image](https://user-images.githubusercontent.com/48056716/161384677-cc744e63-45ef-414d-b96e-deb9cebf8fe3.png)

Reply:
Also the "see detail" button doesn't seem to be doing anything. When we click on it, maybe redirect it to the calendar page. Right now it's just refreshing the page.

Reply:
> Also the "see detail" button doesn't seem to be doing anything. When we click on it, maybe redirect it to the calendar page. Right now it's just refreshing the page.

I just realized that it's refreshing the page because I am using a business account and only employees have access to the calendar page. @nasaku898 Should we not display the calendar in the home page if the user logged in is not an employee?

Reply:
> t

Yeah with the current implementation, we should just limit it to employee as it is not functional for other type of user.

Reply:
@bowsong Only show the calendar if the user is an employee. Otherwise, expand the other grids to take the place of the calendar.

Reply:
Removed comments, added a max height to the chart and made calendar conditional